lib/src/platform/src/realtime/realtime_channel.dart

```diff
@@ -98,6 +98,9 @@ class RealtimeChannel extends PlatformObject {
   /// (see IDL for more details)
   late PushChannel push;
 
+  /// Current state of the channel
+  ///
```

Comment:
Do we need this empty line here? :wink:

Reply:
Yes, the convention tor DartDoc here looks like that:

```
  /// Detaches the realtime client from this channel.
  ///
  /// https://docs.ably.com/client-lib-development-guide/features/#RTL5
```

So there's an empty line between field descriptions and links to external sources 🙂